examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/ConfigureFragment.kt

```diff
@@ -145,6 +145,7 @@ class ConfigureFragment : Fragment() {
             .diagnosticsEnabled(true)
             .entitlementVerificationMode(entitlementVerificationMode)
             .purchasesAreCompletedBy(purchasesAreCompletedBy)
+            .pendingTransactionsForPrepaidPlansEnabled(true)
```

Comment:
I noticed now you couldn't test this in the purchase tester app easily. I think this should be enabled here, so I sneaked it into this PR :P